### DISCUSSION TITLE
Scarving and cutting

### DISCUSSION THREAD

**Discussion by CBYTLSE** [arkypita/LaserGRBL#2160] [Q&A]
Hi,
I just have placed an order for a laser engraver (LONGER RAY5 10W) and use for some years a 3d Printer (Creality Ender 3 Pro). This domain is new for me and I try to discover the GRBL software. Perhaps my question has soon got an answer.
Is it possible to generate a unique gcode file containing a first pass of engraving  and followed by a second pass of cutting?
And how to do it?
Regards

**Comment by arkypita** [comment]
Menu file, append

**Comment by Tribo-2dot0** [comment]
Veja esses dois vídeos, talvez ajude...
www.youtube.com/watch?v=LTU_xWh286I
www.youtube.com/watch?v=k7LZk5Qr7zI

**Reply by CBYTLSE** [reply]
Thank you all. Exactly the videos I need.

All the best.
